boot/makebootable.go

```diff
@@ -78,6 +78,19 @@ func MakeBootableImage(model *asserts.Model, rootdir string, bootWith *BootableS
 	return makeBootable20(rootdir, bootWith, bootFlags)
 }
 
+// MakeBootableOnTarget configures a partition mounted on rootdir
+// using information from bootWith and bootFlags. Contrarily to
+// MakeBootableImage this happens in a live system.
+func MakeBootableOnTarget(rootdir string, bootWith *BootableSet, bootFlags []string) error {
+	opts := &bootloader.Options{
+		PrepareImageTime: true,
```

Comment:
I'm not sure if this makes sense or not for runtime. I don't remember the bootloader behavior controlled by this.